src/ThirdTask.java

```diff
@@ -0,0 +1,40 @@
+public class ThirdTask {
+    public static int absoluteDifference(int a, int b) {
+        return Math.abs(a - b);
+    }
+    public static char getFirstChar(String str) {
+        return str.charAt(0);
+    }
+public static char getLastChar(String str) {
+    return str.charAt(str.length() - 1);
+
+
+
+
+
+
+}
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+        int result = absoluteDifference(a, b);
+        System.out.println("a = " + a);
+        System.out.println("b = " + b);
+        System.out.println("result = " + result);
+
+        String inputString = "Python";
+        char firstChar = getFirstChar(inputString);
+        System.out.println("string = " + inputString );
+        System.out.println("result = " + firstChar );
+
+
+
+char lastChar = getLastChar(inputString);
+System.out.println("result = " + lastChar);
+
+String substring = inputString.substring(2 ,6);
+System.out.println(lastChar + substring + firstChar);
```

Comment:
Please move it to the method as well

Reply:
Done

Reply:
It is still in a method, you just removed formatting. 
Make a new method
public String swapFirstAndLastCharacter(String string) {
// your code here
}